### DISCUSSION TITLE
Enabling new translations inside app

### DISCUSSION THREAD

**Discussion by Klem7ens** [TandoorRecipes/recipes#964] [Q&A] [answered]
Hi,
I wanted to check/test Polish translation, but can't select this language in Settings, no such option on drop down list. Do I have to set something in .env file or this must be set in app code?

**Comment by vabene1111** [comment, marked as answer]
i am sorry, i did not realise polish was new in this release. I will add it to the selection so you can have it for the next release which will likely be out soon

**Reply by Klem7ens** [reply]
Its not a problem to wait some more. Thx for quick answer, because thought that I'm doing something wrong. :)
